src/categorical_algebra/DataMigrations.jl

```diff
@@ -319,9 +319,8 @@ function (M::SigmaMigrationFunctor)(d::ACSet; n=100, return_unit::Bool=false)
   for (k,v) in datatypes(D)  atypes[Symbol(ob_map(i1,k))] = v end
   for (k,v) in datatypes(CD) atypes[Symbol(ob_map(i2,k))] = v end
   col_type = crel_type(presentation(apex(col)); types=atypes, name="Sigma")()
-  for o in ob(S)
+  for o in types(S)
     add_parts!(col_type, Symbol(ob_map(i1,o)), nparts(d,o))
-    #add parts for attrvars?
```

Comment:
The answer was "Yes". :)

Reply:
Haha, great! Thanks for the fix.

Reply:
OK, fixed in a mathematically-correct, code-unprincipled way by borrowing @kris-brown's function for stripping free variables from AlgebraicRewriting. He said he might have a minute to think about how that function should really be rewritten and where it should go (in ACSets.jl, I suppose) but I'll do that if he doesn't get to it.

Reply:
Why is `remove_freevars` needed here? The chase shouldn't introduce any free variables.

Reply:
When you migrate up to the collage, all the attributes are initially valued as attrvars, and then it's post-chase that we go back and re-impose naturality on attributes. That can leave some of those original attrvars floating.